portfolio/src/main/webapp/script.js

```diff
@@ -16,9 +16,13 @@
  * Fetches comments and appends them as list elements.
  */
 function getComments() {
-  fetch('/data').then(response => response.text()).then((comments) => {
+  const commentLimit = document.getElementById('limit-comments').value;
+  fetch('/data?limit-comments=' + commentLimit)
+  .then(response => response.text())
```

Comment:
Indent this 4 spaces, again, a continued long line. Match the following `.then` with it.